libs/shared/src/lib/models/asset-edit.ts

```diff
@@ -110,3 +110,8 @@ export const AssetEditDetail = C.struct({
   studies: C.array(C.struct({ assetId: C.number, studyId: C.string, geomText: C.string })),
 });
 export type AssetEditDetail = C.TypeOf<typeof AssetEditDetail>;
+
+// todo: once AssetEditDetail is removed, a similar property could exist on the new model to avoid this helper method
+export const hasHistoricalData = (asset: AssetEditDetail) => {
+  return asset.geolAuxDataInfo || asset.geolContactDataInfo || asset.geolDataInfo || asset.municipality || asset.sgsId;
+};
```

Comment:
May be nice to make this type safe.
```suggestion
export const hasHistoricalData = (asset: AssetEditDetail): boolean => {
  return !!(asset.geolAuxDataInfo || asset.geolContactDataInfo || asset.geolDataInfo || asset.municipality || asset.sgsId);
};
```